l1-contracts/src/core/Rollup.sol

```diff
@@ -24,12 +24,14 @@ import {MockVerifier} from "@aztec/mock/MockVerifier.sol";
 contract Rollup is IRollup {
   MockVerifier public immutable VERIFIER;
   IRegistry public immutable REGISTRY;
+  uint256 public immutable VERSION;
 
   bytes32 public rollupStateHash;
 
   constructor(IRegistry _registry) {
     VERIFIER = new MockVerifier();
     REGISTRY = _registry;
+    VERSION = 1;
```

Comment:
should this be in the constructor so it is easier to not miss updating this var when upgrading?

Reply:
Would depend on the upgrade mechanism, and since we are not sure about that yet, have just put it like this.